templates/definition/tariff/elering.yaml

```
@@ -1,19 +1,19 @@
-template: elering
+template: Nordpool
 products:
-  - brand: Nordpool
+  - brand: Nordpool-Group
     description:
-      generic: "Elering"
+      generic: "Nordpool"
 requirements:
   description:
-    de: "Nur für Estland verfügbar."
-    en: "Only available for Estonia."
+    de: "Verfügbare Regionen sind Estland, Lettland, Litauen, Finnland, Schweden, Norwegen, Dänemark, Rumänien, Österreich, Belgien, Frankreich, Deutschland, Niederlande, Polen."
+    en: "Available regions are Estonia, Latvia, Lithuania, Finland, Sweden, Norway, Denmark, Romania, Austria, Belgium, France, Germany, Netherlands, Poland."
 group: price
 params:
   - name: region
     example: ee
-    validvalues: ["ee", "lt", "lv", "fi"]
+    validvalues: ["at", "be", "dk1", "dk2", "ee", "lt", "lv", "fi", "fr", "ger", "nl", "no1", "no2", "no3", "no4", "no5" ,"pl", "se1", "se2", "se3", "se4", "tel" ]
   - preset: tariff-base
 render: |
-  type: elering
+  type: Nordpool
```

```suggestion
  type: elering
```

there is no "Nordpool" Go code ;)

---

This looks strange to me, as Elering only provides data for the 4 countries (FI, EE, LV, LT) 
???